examples/test_tsd_syn.py

```diff
@@ -0,0 +1,175 @@
+# [THINSHELLDEMONS] Read Mesh and Create TSD object
```

Comment:
There is quite a bit of commonality with affine case. Can they be merged, with some
```py
if affine:
  ...
else:  # SyN
  ...
```

Reply:
Yes, it could be done. 
However, personally I prefer individual files to avoid complexity in understanding.

I have followed the style similar to C++ so that user can easily find the counterpart in Python. There is another DisplacementFieldTransform example and it is very close to Affine Example. 
The Syn Registration had more difference so I added that first. 

And I removed the part of DisplacementFieldTransformAdaptorType present in the C++ code in the Python version.
That part, I think was added to demonstrate that it could be used too but in the code the result was not impacted due to it. So I have omitted that part in Python.

Reply:
OK